build.gradle

```diff
@@ -182,6 +182,11 @@ subprojects {
 
   task docsJar(dependsOn: javadocJar)
 
+  javadoc {
+    options.charSet = 'UTF-8'
+    options.encoding = 'UTF-8'
```

Comment:
Within this javadoc block, does this cover everything we need? It seems options is a https://docs.gradle.org/current/javadoc/org/gradle/external/javadoc/StandardJavadocDocletOptions.html and it also has a `docEncoding` value, which based on Java documentation seems to specify the output encoding whereas `encoding` specifies the input. Do we need both, or were you intentionally only setting the input encoding? I think since `charSet` is written to the HTML output, we'd want `charSet` and `docEncoding` to be identical.

Also, what is the right way to test this. Does something like setting your locale to certain values produce the garbled output?